tests/util.test.js

```diff
@@ -294,16 +301,16 @@ describe('pmd-github-action-util', function () {
     process.env['GITHUB_EVENT_NAME'] = 'push';
     process.env['GITHUB_EVENT_PATH'] = __dirname + '/data/push-event-data.json';
     nock('https://api.github.com')
-      .get('/repos/pmd/pmd-github-action-tests/compare/44c1557134c7dbaf46ecdf796fb871c8df8989e4...8a7a25638d8ca5207cc824dea9571325b243c6a1?per_page=30&page=1')
+      .get('/repos/pmd/pmd-github-action-tests/commits/8a7a25638d8ca5207cc824dea9571325b243c6a1?per_page=30&page=1')
       .replyWithFile(200, __dirname + '/data/compare-files-page1.json', {
```

Comment:
Please also rename this file "compare-files-page1.json" and update the contents - it's now not anymore the response from compare, but from commits, which potentially looks different.

You can just look at https://api.github.com/repos/pmd/pmd-github-action-tests/commits/8a7a25638d8ca5207cc824dea9571325b243c6a1?per_page=30&page=1 to get the json.

Note: I've simplified the original response for compare back then to make it smaller and contain only the essential parts - and modified the files part to contain the data I needed for tests.